packages/plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -261,6 +262,72 @@ describe(`plugin-conversation`, function() {
           assert.lengthOf(activity.object.mentions.items, 1);
           assert.equal(activity.object.mentions.items[0].id, mccoy.id);
         }));
+
+      skipInNode(describe)(`when there is html tags in rich messages`, () => {
+        [
+          {
+            it: `allows allowed outbound and inbound tags`,
+            allowedOutboundTags: {div: []},
+            allowedInboundTags: {div: []},
+            outboundMessage: `<div>HELLO</div>`,
+            outboundFileredMessage: `<div>HELLO</div>`,
+            inboundMessage: `<div>HELLO</div>`
+          },
+          {
+            it: `filters disallowed outbound tags`,
+            allowedOutboundTags: {},
+            allowedInboundTags: {},
+            outboundMessage: `<div><b>HELLO</b></div>`,
+            outboundFileredMessage: `HELLO`,
+            inboundMessage: `HELLO`
+          },
+          {
+            it: `filters disallowed inbound tags`,
+            allowedOutboundTags: {div: [], b: []},
+            allowedInboundTags: {b: []},
+            outboundMessage: `<div><b>HELLO</b></div>`,
+            outboundFileredMessage: `<div><b>HELLO</b></div>`,
+            inboundMessage: `<b>HELLO</b>`
+          },
+          {
+            it: `filters the correct outbound tags`,
+            allowedOutboundTags: {div: [], span: []},
+            allowedInboundTags: {},
+            outboundMessage: `<div><b>HELLO</b><span> it's me</span></div>`,
+            outboundFileredMessage: `<div>HELLO<span> it's me</span></div>`,
+            inboundMessage: `HELLO it's me`
+          },
+          {
+            it: `filters the correct inbound and outbound tags`,
+            allowedOutboundTags: {div: [], span: []},
+            allowedInboundTags: {span: []},
+            outboundMessage: `<div><b>HELLO</b><span> it's me</span></div>`,
+            outboundFileredMessage: `<div>HELLO<span> it's me</span></div>`,
+            inboundMessage: `HELLO<span> it's me</span>`
+          }
+        ].forEach((def) => {
+          it(def.it, () => {
+            // use same rules for user 1 so that we know the message is altered
+            // as desired
+            spark.config.conversation.allowedOutboundTags = def.allowedOutboundTags;
+            spark.config.conversation.allowedInboundTags = def.allowedOutboundTags;
```

Comment:
I've added the comments.
What I'm thinking is that if we let allowedInboundTags to allow all the tags (kind of like disable inbound filter) then we know our messages are filtered by the outbound rules. But this is assuming inbound filter is doing what it's supposed to do.
I wonder if I should split the tests into two:
- outbound/inbound for spock
- inbound for mccoy (to test with another user)